drivers/windows/overlay/ov_endpoint_windows.go

```diff
@@ -271,11 +271,11 @@ func (d *driver) EndpointOperInfo(nid, eid string) (map[string]interface{}, erro
 }
 
 func endpointRequest(method, path, request string) (*hcsshim.HNSEndpoint, error) {
-	if windowsBuild == 14393 {
+	if windowsBuild == osversion.RS1 {
```

Comment:
Also wondering if this should be 

```suggestion
	if windowsBuild < osversion.RS3 {
```

(assuming RS2 is EOL / obsolete)

Reply:
Either is fine since RS2 is obsolete.
I have a slight preference to "windowsBuild == osversion.RS1"  since that is what we investigated and validated.

Reply:
Per the discussion on https://github.com/Microsoft/hcsshim/pull/556#discussion_r275547634, this should likely be more defensive and take anything < RS5

```suggestion
	if windowsBuild < osversion.RS5 {
```

Reply:
LGTM.